### ISSUE TITLE
Low Accuracy Than Reported

### ISSUE THREAD

**Issue by wangtuo13** [Yunfan-Li/Contrastive-Clustering#12]
I run the code in win10+python3.7+pytorch1.8.1+cu111 and get 21% accuracy on Cifar10 dataset.

**Comment by Yunfan-Li** [comment]
How many epochs have you trained?

**Comment by mouxingyang** [comment]
from simon

**Comment by haoweiclouds1** [comment]
i also have this problem , i trained with 1000 epoch as the same as you. please give some suggestion,thank you very much!

**Comment by haoweiclouds1** [comment]
results as follows: ( run cluster.py)
Features shape (60000,)
NMI = 0.1276 ARI = 0.0685 F = 0.1795 ACC = 0.2311

**Comment by Yunfan-Li** [comment]
Please make sure that you reload the model trained for 1000 epoch. Perhaps the model at epoch 0 is loaded for clustering.

**Comment by haoweiclouds1** [comment]
thank you i will try again

**Comment by haoweiclouds1** [comment]
sorry ,I return to ask you some questions.

I have trained for 1000 epoch and the model has saved,

how can i load in cluster.py in your code 
(eg :i find the path i have saved : Contrastive-Clustering-main/save/CIFAR-10/checkpoint_990.tar)

thank you very much!

**Comment by Yunfan-Li** [comment]
Changing the ''start epoch'' number in the config file to 990 will do.

**Comment by haoweiclouds1** [comment]
hh  thank you ,i find !

**Comment by haoweiclouds1** [comment]
your work is very wonderful!!

**Comment by Yunfan-Li** [comment]
Thank you!

**Comment by haoweiclouds1** [comment]
yunfan 您好，我用990 试了，acc大约在0.5左右，排查了代码 ，实在没发现问题，您还有什么建议吗 。感谢你的耐心解答！

**Comment by Yunfan-Li** [comment]
您好，可以先试一下用release的模型测试看下结果是否正确，然后确保您没有改动训练的代码和config中的超参数，重新训练看下结果，因为软硬件环境的不同，根据目前的反馈可能有1-2个点的偏差，但是0.5一定是哪里出了问题。

**Comment by haoweiclouds1** [comment]
好的 我试试 感谢
